airflow/providers/singularity/operators/__init__.py

```diff
@@ -0,0 +1,172 @@
+#
```

Comment:
We should avoid adding code in `__init__.py` . Each operator should have their own file. Adding code to `__init__.py` causes the side effect that the code is imported and the imports are further processed when the package is imported. This is very bad as it is implicit behavior not expected by the one who imports it and it causes all the various problems with chaining loading of unnecessary imports which slows down application startup. We are just now voting in the devlist on removal of `airflow.__init__.py` as this is very bad example on how things should not be done.

Please move it to separate class - this should also solve the problem with documentation building. One of the side effects of adding imports to __init__.py is tha the imported AirflowException class is now imported in two places.

Reply:
Sure, happy to do this - it looks like it's the same case for the testing files, so I'll follow suit there.